pkg/bot/interactive/help.go

```diff
@@ -182,7 +182,7 @@ func (h *HelpMessage) configSections() []Section {
 
 func (h *HelpMessage) kubectlSections() []Section {
 	// TODO(https://github.com/kubeshop/botkube/issues/802): remove this warning in after releasing 0.17.
```

Comment:
Will we remember to remove this prefix after 0.17? Or can we remove it even now? 🤔

Reply:
![tenor-257224583](https://user-images.githubusercontent.com/7155799/210549367-a98496e8-c606-403b-9c3c-75b85945192e.gif)